src/gpu/intel/jit/gemm/generator/pieces/address_setup.cxx

```diff
@@ -216,7 +216,7 @@ void BLASKernelGenerator<hw>::setupAddr(Type T, const GRFRange &addr, const BO &
                 }
 
                 if (a64) {
-                    int udStride = (hw >= HW::XeHP) ? 2 : 1;
+                    constexpr int udStride = (hw >= HW::XeHP) ? 2 : 1;
```

Comment:
Just curious, what is Coverity complaining about? Would `const` make it happy? (If so, I think that makes more sense than `constexpr` here...)